webapp/frontend/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Can we set target to ES6?

The following link provides comparison between ES5 and ES6
https://www.javatpoint.com/es5-vs-es6#:~:text=Difference%20between%20ES5%20and%20ES6&text=ES5%20supports%20primitive%20data%20types,symbol'%20for%20supporting%20unique%20values.

https://www.geeksforgeeks.org/difference-between-es5-and-es6/

Reply:
agreed. I missed to make the change here